packages/lit-node-client-nodejs/src/lib/helpers/session-sigs-validator.ts

```diff
@@ -0,0 +1,210 @@
+import {
+  AuthSig,
+  Capability,
+  ParsedSessionMessage,
+  ParsedSignedMessage,
+  SessionSigsMap,
+} from '@lit-protocol/types';
+
+interface ValidationResult {
+  isValid: boolean;
+  errors: string[];
+}
+
+// Function to parse a signedMessage string into an object
+function parseSignedMessage(signedMessage: string): ParsedSignedMessage {
```

Comment:
Is there a reason we cannot use the `SiweMessage` constructor to parse?

Reply:
ahh good catch! could've been easier.. will change that

Reply:
following up - actually, `capabilities` is a custom property that is not part of the standard `SIWEMessage` specification, so it will always throw an error. I'm leaving it as before.

Reply:
Actually it's a bit tricky, the `capabilities` field is not part of the standard `SIWEMessage` specification, instead it is  part of EIP-5573: SIWE ReCap which extends it, so using `SiweMessage` will always throw an error.

Reply:
Ah I see, that is annoying, we do have a fork of the Library [here](https://github.com/LIT-Protocol/siwe). We might want to follow up in the future by adding functionality to our fork to make this helper cleaner.